Packs/Sixgill-Darkfeed/pack_metadata.json

```
@@ -2,18 +2,54 @@
     "name": "Sixgill Darkfeed - Annual Subscription",
     "description": "This edition of Sixgill Darkfeed is intended for customers who have a direct annual subscription to Sixgill Darkfeed.\n\nGet contextual and actionable insights to proactively block underground threats in real-time with the most comprehensive, automated stream of IOCs \n\nFor organizations who are currently Darkfeed customers.",
     "support": "partner",
-    "currentVersion": "1.2.1",
+    "currentVersion": "1.2.2",
     "author": "Sixgill",
-    "url": "https://info.cybersixgill.com/darkfeed-demo-request?__hstc=243871258.1ada04342b527588fbfc3ce61f48de94.1595252142770.1595252142770.1595254236156.2&__hssc=243871258.1.1595254236156&__hsfp=3682459443",
-    "email": "support@cybersixgill.com",
+    "url": "",
+    "email": "sales@cybersixgill.com",
     "created": "2020-03-10T08:37:18Z",
     "categories": [
-        "Data Enrichment & Threat Intelligence"
+        "Data Enrichment & Threat Intelligence",
+        "Forensics & Malware Analysis"
+    ],
+    "tags": ["threat intelligence", "Dark Web", "Malware"],
```

```suggestion
    "tags": ["Threat Intelligence", "Dark Web", "Malware"],
```
Let's capitalize this